<ui version="4.0" >
 <class>PayStatusDialog</class>
 <widget class="QDialog" name="PayStatusDialog" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>266</width>
    <height>202</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Подтверждение оплаты</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>4</number>
   </property>
   <property name="topMargin" >
    <number>4</number>
   </property>
   <property name="rightMargin" >
    <number>4</number>
   </property>
   <property name="bottomMargin" >
    <number>4</number>
   </property>
   <property name="horizontalSpacing" >
    <number>4</number>
   </property>
   <property name="verticalSpacing" >
    <number>4</number>
   </property>
   <item row="2" column="1" colspan="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>201</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" >
    <widget class="QLineEdit" name="edtNumber" >
     <property name="maxLength" >
      <number>20</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QRadioButton" name="rbnAccepted" >
     <property name="text" >
      <string>Оплачено</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="lblNumber" >
     <property name="text" >
      <string>Документ</string>
     </property>
     <property name="buddy" >
      <cstring>edtNumber</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="lblDate" >
     <property name="text" >
      <string>Дата</string>
     </property>
     <property name="buddy" >
      <cstring>edtDate</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QRadioButton" name="rbnRefused" >
     <property name="text" >
      <string>Отказано</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QLabel" name="lblPayRefuseType" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>Причина отказа</string>
     </property>
     <property name="buddy" >
      <cstring>cmbRefuseType</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>81</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>81</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1" colspan="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>201</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0" colspan="3" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2" >
    <widget class="CRBComboBox" name="cmbRefuseType" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="CDateEdit" name="edtDate" />
   </item>
   <item row="5" column="1" colspan="2" >
    <widget class="QLineEdit" name="edtNote" />
   </item>
   <item row="5" column="0" >
    <widget class="QLabel" name="lblNote" >
     <property name="text" >
      <string>Примечание</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit.h</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtDate</tabstop>
  <tabstop>edtNumber</tabstop>
  <tabstop>rbnAccepted</tabstop>
  <tabstop>rbnRefused</tabstop>
  <tabstop>cmbRefuseType</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PayStatusDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>227</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel" >
     <x>153</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PayStatusDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>235</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel" >
     <x>244</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbnRefused</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbRefuseType</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>45</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel" >
     <x>103</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbnRefused</sender>
   <signal>toggled(bool)</signal>
   <receiver>lblPayRefuseType</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
